change_ttl/tests/analytics_hits.test.result

```diff
@@ -0,0 +1,2 @@
+"total"
+1000
```

Comment:
I don't understand this test. In a realistic environment you're continuously ingesting data so you can't rely on an static count. I would try with fixtures or even quality tests depending on what you want to check.

Reply:
Right, what I was aiming to test here is if you haven't deleted the data, which could happen when you change the TTL if you add a wrong condition. I'll count specific values to ensure they're still there.

Reply:
Fixed 👍🏽